examples/docs_snippets/docs_snippets/deploying/executors/executors.py

```diff
@@ -37,11 +37,11 @@ def op_job():
     ...
 
 
-# op_job and asset_job will both use the default_executor_def,
+# op_job and asset_job will both use the multiprocess_executor,
 # since neither define their own executor.
-@repository(default_executor_def=multiprocess_executor)
-def the_repo():
-    return [the_asset, asset_job, op_job]
 
+defs = Definitions(
+    assets=[the_asset], jobs=[asset_job, op_job], executor=[multiprocess_executor]
```

Comment:
```suggestion
    assets=[the_asset], jobs=[asset_job, op_job], executor=multiprocess_executor
```

Reply:
🤦‍♀️  that explains the `list` error. Thank you!